src/components/itemContextMenu.js

```diff
@@ -11,6 +11,23 @@ import ServerConnections from './ServerConnections';
 import toast from './toast/toast';
 import * as userSettings from '../scripts/settings/userSettings';
 
+function getDeleteLabel(type) {
+    switch (type) {
+        case 'Series':
```

Comment:
Please use `BaseItemKind` from the SDK for these values.

```js
import { BaseItemKind } from '@jellyfin/sdk/lib/generated-client/models/base-item-kind';
```

Reply:
Nice, I didn't know it existed.
Done